emscripten/emscripten_runner.js

```diff
@@ -0,0 +1,118 @@
+const { opendir } = require('node:fs/promises');
```

Comment:
I think ideally we should ship this setup in https://github.com/pyodide/pyodide/tree/main/pyodide-test-runner so one could just pip install it and not have this file in each tested repo.